src/main/resources/_config.properties

```diff
@@ -110,4 +110,11 @@ crypto_key_size=128
 crypto_key_path=./src/main/resources/crypto.key
 #=====================================================#
 password={crypt:8O9iA4+f3nMzz85szmvKmQ==}
-credentials=test@gmail.com/${password}
\ No newline at end of file
+credentials=test@gmail.com/${password}
+
+#================ Locale Configurations ====================#
```

Comment:
Please move locale settings upper. Between "Environment configuration" and "WebDirver configuration"